src/Twigc/DefaultCommand.php

```diff
@@ -205,6 +211,7 @@ public function doRender(InputInterface $input, OutputInterface $output) {
 		$temp      = false;
 		$strict    = (bool) $input->getOption('strict');
 		$escape    = $input->getOption('escape');
+		$wantEnv   = $input->getOption('env');
```

Comment:
This doesn't seem necessary, right? We can just check the option value directly down below